config/services.go

```diff
@@ -183,6 +183,8 @@ func (s *AppServices) loadBux(ctx context.Context, appConfig *AppConfig, testMod
 		options = loadBroadcastClientArc(appConfig, options, logger)
 	}
 
+	options = append(options, bux.WithCallback(appConfig.Callback.CallbackHost+BroadcastCallbackRoute, appConfig.Callback.CallbackToken))
```

Comment:
Maybe this would be a good place to check if CallbackToken is not empty. If not, then we can generate it from admin xpub here.

Reply:
done